.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+### Description
+What does this do? What problem/s does it solve? What issue numbers does it
+resolve?
+
+### Checklist
+- [ ] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
+- [ ] My patch is based on the correct branch (eg. `6.6.x` or `master`)
+- [ ] I am opening this PR against that branch
```

Comment:
We haven't accepted PRs against `6.6.x` in several months. These two items are probably wholly unnecessary; we can handle patches that need to be made against a maintenance branch on a case-by-case basis.

Reply:
I was trying to come up with a concise way to say "If this is a bugfix, find the version's branch", but eh if you don't mind the additional handholding and people having to go rebase stuff.